scripts/start_production_postgres.sh

```diff
@@ -30,7 +30,7 @@ POSTGRES_ROOT="/var/lib/postgresql/14"
 
 
 PROJECT_DIR=`pwd`
-PRODUCTION_CLUSTER_DIR="${POSTGRES_ROOT}/production_cluster"
+PRODUCTION_CLUSTER_DIR="${PROJECT_DIR}/production_cluster"
```

Comment:
now that we're using PROJECT_DIR to create production cluster, I guess the sample command in line 6 (sudo -u postgres...) will be invalid cuz user postgres can't mkdir in PROJECT_DIR

Reply:
No this is an unintentional push, will revert